lib/bundler/templates/newgem/test/test_newgem.rb.tt

```diff
@@ -0,0 +1,14 @@
+$LOAD_PATH.unshift File.expand_path("../lib", __dir__)
+require "<%= config[:namespaced_path] %>"
+
+require "test/unit"
+
+class Test<%= config[:constant_name] %> < Test::Unit::TestCase
+  def test_that_it_has_a_version_number
+    refute_nil ::<%= config[:constant_name] %>::VERSION
```

Comment:
The following code is test-unit style:

```ruby
  def test_version
    assert do
      ::<%= config[:constant_name] %>.const_defined?(:VERSION)
    end
  end
```

See also: https://github.com/bundler/bundler/pull/5521/files#diff-7675ffc4b1b026242f4099e54c032b7b